testing-mysql-server-base/src/main/java/com/facebook/presto/testing/mysql/AbstractTestingMySqlServer.java

```diff
@@ -50,8 +53,8 @@ public AbstractTestingMySqlServer(EmbeddedMySql server, String user, String pass
         try (Connection connection = server.getMySqlDatabase()) {
             version = connection.getMetaData().getDatabaseProductVersion();
             try (Statement statement = connection.createStatement()) {
-                execute(statement, format("CREATE USER '%s'@'%%' IDENTIFIED BY '%s'", user, password));
-                execute(statement, format("GRANT ALL ON *.* to '%s'@'%%' WITH GRANT OPTION", user));
+                execute(statement, format("CREATE USER '%s'@'%s' IDENTIFIED BY '%s'", user, isMariadb ? "localhost" : "%%", password));
```

Comment:
don't they both run on localhost

Reply:
Yes, they do. But mariadb seems to not consider `%%` equivalent to `localhost` causing the tests to fail for **presto-function-namespace-managers** package with error:
`Caused by: java.sql.SQLException: Access denied for user 'testuser'@'localhost' (using password: YES)`

Reply:
since localhost works for both mysql and mariadb, i would just change this to use localhost, rather than make it conditional

Reply:
Right, will do.